fastcore/meta.py

```diff
@@ -120,20 +120,21 @@ def _f(f):
         k = sigd.pop('kwargs')
         s2 = {k:v for k,v in inspect.signature(to_f).parameters.items()
```

Comment:
```suggestion
        s2 = {k:v.replace(kind=inspect.Parameter.KEYWORD_ONLY) for k,v in inspect.signature(to_f).parameters.items()
```

If you made this change, would that mean you could then remove the line you added?

Reply:
Yes, this change works and my added line can be removed. Thanks Jeremy!

Reply:
Great. Just reply here when you've removed the `_docs` folder and we should be ready to go!